docs/installation.md

```diff
@@ -79,6 +79,20 @@ This option will hard reset your branch (only if you are on either `master` or `
 
 DEPRECATED - use `freqtrade new-config -c config.json` instead.
 
+### MacOS installation error
```

Comment:
I would move this section to the bottom of this document.

It's trouble-shooting, really - and most likely irrelevant to most users.

Reply:
Done. In the future this could be a footnote. It looks weird that after we present the conclusion of the page and direct the user to another page we are adding troubleshooting.

Reply:
yeah, i've moved this o top of the footnote now - it used to be a "troubleshooting" section ...